.github/workflows/update-kernels.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Kernels
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * 2'
```

Comment:
This should be a weekly job, like: `0 2 * * 1` (ie: At 02:00 on Monday.)

Reply:
Got it, thanks